source/GameInterface/GameInterface.csproj

```diff
@@ -216,4 +216,8 @@
     <PackageReference Include="Bannerlord.Lib.Harmony" Version="2.2.2" />
     <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
   </ItemGroup>
+  <ItemGroup>
+    <Folder Include="Services\Encounters\Patches\" />
+    <Folder Include="Services\MapEvents\Handlers\" />
+  </ItemGroup>
```

Comment:
These don't exist for any other folder, try removing these and looking through you settings and disabling this